src/models/licensemodel.cpp

```diff
@@ -60,8 +60,8 @@ void LicenseModel::initialize() {
 
   beginResetModel();
 
-  for (uint32_t i = 0, length = content.length(); i < length; ++i) {
-    int32_t lineLength = content[i].length();
+  for (qsizetype i = 0, length = content.length(); i < length; ++i) {
+    qsizetype lineLength = content[i].length();
```

Comment:
Is this change actually related to the CheckedInt stuff or is it just a case of some implicit type coersion was going on before and is not anymore?

Reply:
I removed a bunch of int warnings too.

Reply:
Ah! Got it? Should we be failing CI on warning to avoid having things like these in the future?

Reply:
These warnings are shown only on iOS...

Reply:
Sheesh, ok yeah let's forget about it now then 👀